lib/types/settings.ak

```diff
@@ -88,6 +87,19 @@ pub type SettingsRedeemer {
 /// The name of the token that authenticates the settings UTXO
 pub const settings_nft_name: AssetName = "settings"
 
+pub fn find_protocol_fee_extension(
```

Comment:
since the first thing we do is fail when this isn't set (via `expect Some(...)`), we could probably just name this `must_find_protocol_fee_extension` and simplify the code.

but it's not a major deal.